src/onboarding/pythonReqsManager.ts

```diff
@@ -72,7 +72,7 @@ export async function checkPythonRequirements(
   process.env.IDF_PATH = espIdfPath || process.env.IDF_PATH;
   await utils
     .execChildProcess(
-      `${pythonBin} ${reqFilePath} -r ${requirements}`,
+      `"${pythonBin}" ${reqFilePath} -r ${requirements}`,
```

Comment:
`reqFilePath ` and `requirements ` might also contain spaces, if so, what will happen then?